readalongs/align.py

```diff
@@ -33,46 +33,58 @@
 from readalongs.text.util import save_xml
 from readalongs.log import LOGGER
 
-####
-#
-# Some distros (Python2, Python3 on Windows it seems) don't have the WAV
-# reading methods being context managers; the following checks whether the
-# necessary methods are present and if not, add thems.
-#
-# Based on http://web.mit.edu/jgross/Public/21M.065/sound.py 9-24-2017
-####
-
 
-def _trivial__enter__(self):
-    return self
+def correct_adjustments(start: Union[int, float], end: Union[int, float], do_not_align_segments: List[object]) -> (int, int):
```

Comment:
Now that this function is just in ms, do we still need `Union[int, float]`? The Black doc suggests that this is equivalent to just `float`, so I would switch it, as long as the code still works when we do.